docs/spec/api.md

```diff
@@ -1108,7 +1133,9 @@ A list of methods and URIs are covered in the table below:
 |Method|Path|Entity|Description|
 |------|----|------|-----------|
 | GET | `/v2/` | Base | Check that the endpoint implements Docker Registry API V2. |
-| GET | `/v2/<name>/tags/list` | Tags | Fetch the tags under the repository identified by `name`. |
+| GET | `/v2/<name>/tags/_list` | Tags | Fetch the tags under the repository identified by `name`. |
+| GET | `/v2/<name>/tags/<reference>` | Tags | Redirect to tags list. |
```

Comment:
For `DELETE` sure, this is `GET`.  Your change description specifically mentions redirecting `/tags/list` to `/tags/_list`:

> It introduces new TagAPI which allows deleting tags for a given image and also supports legacy Tags List URL /v2/<name>/tags/list by redirecting it to the new one /v2/<name>/tags/_list.

So I am a bit confused by this line.